pants-plugins/internal_plugins/releases/register.py

```diff
@@ -131,10 +131,9 @@ async def check_default_tools(
                     SessionValues(
                         {
                             OptionsBootstrapper: OptionsBootstrapper(
-                                tuple(),
-                                ("./pants",),
-                                args,
-                                Config(tuple()),
+                                args=args,
+                                env=FrozenDict(),
```

Comment:
Note that the type of `env` was changed be a mapping instead of the less obvious list of pairs.